xonsh/xoreutils/which.py

```diff
@@ -97,7 +97,10 @@ def which(args, stdin=None, stdout=None, stderr=None, spec=None):
 
     pargs = parser.parse_args(args)
     verbose = pargs.verbose or pargs.all
-    captured = spec.captured in xproc.STDOUT_CAPTURE_KINDS
+    if spec is not None:
+        captured = spec.captured in xproc.STDOUT_CAPTURE_KINDS
+    else:
+        captured = False
```

Comment:
I think these do the same thing since None is not in `xproc.STDOUT_CAPTURE_KINDS`, right?